Gemfile.lock

```diff
@@ -120,8 +120,8 @@ GEM
     bullet (6.0.2)
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
-    byebug (11.1.1)
-    capistrano (3.12.1)
+    byebug (11.1.3)
```

Comment:
### [byebug](https://github.com/deivid-rodriguez/byebug)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 11.1.1 → 11.1.3

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/byebug)_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/byebug/compare/v11.1.1...v11.1.3).

These are the first 10 commits:
- (161c098) [Rename build workflow to ubuntu](https://github.com/deivid-rodriguez/byebug/commit/161c098bf2d5f33e5fd6e03595429a7e780501ad)
- (048d83f) [Rename fedora's workflow main job](https://github.com/deivid-rodriguez/byebug/commit/048d83ff00b514a95622f77257cce8d19770c98f)
- (1335bde) [Rename daily workflow and main job](https://github.com/deivid-rodriguez/byebug/commit/1335bde2326911d81f91a80121b2f61a3080952c)
- (3519235) [Merge pull request #639 from deivid-rodriguez/ci/renames](https://github.com/deivid-rodriguez/byebug/commit/3519235a29fe13f9d5abb71f80e45954a34d41d6)
- (5219777) [Remove unnecessary quoting (#640)](https://github.com/deivid-rodriguez/byebug/commit/52197774bc6da445c4b45df41b65648dbfb2b459)
- (87d9a7e) [Use latest byebug image in daily workflow (#641)](https://github.com/deivid-rodriguez/byebug/commit/87d9a7e382ba93aafb9ccbabc7e9600e00319b3f)
- (ab9bf88) [Simplify shelling out helpers in remote debugging tests](https://github.com/deivid-rodriguez/byebug/commit/ab9bf882e92784045817a05685d62884d21d7431)
- (c08feba) [Avoid passing around the error thread too](https://github.com/deivid-rodriguez/byebug/commit/c08feba9a4ac76d3191a0999bda5a0b50fc5b99b)
- (4e359bd) [Only print errors if command fails](https://github.com/deivid-rodriguez/byebug/commit/4e359bd5f7eecc02ad4193d8455bb9582d2babf6)
- (11d28b9) [Print the whole output of command on errors](https://github.com/deivid-rodriguez/byebug/commit/11d28b9dfcc621cd9227f6a3532b4083aa0cc693)


</details>